services/cognito/app/resources/pool_resource.rb

```diff
@@ -5,13 +5,17 @@ class PoolResource < Cognito::ApplicationResource
   has_many :users
 
   filter :query, apply: lambda { |records, value, _options|
-    query_by_id = "pools.id IN (#{value[0]})"
```

Comment:
@rpbaltazar do we need to be able to pass an array of IDs here?

Reply:
It was supported, I think we should keep the support